packages/frontend/src/components/profile/remove_account/ui/PasswordConfirmationModal/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC, useEffect, useState, useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { currentTargetValue, forkEvent } from '../../../../../shared/lib/forms/selectors';
+import { isPasswordValid } from '../../../../accounts/EnterPasswordForm/lib/validate';
```

Comment:
cross-components import :(

Reply:
But if we'll use the the whole form it won't solve import problems 
`../../../../accounts/EnterPasswordForm`